src/libs/utils.js

```diff
@@ -368,3 +368,5 @@ export const truncateInteger = integer => {
 
   return `${Math.floor(integer / 1000)}k`
 }
+
+export const enumify = keys => _.flow(_.map(key => [key, key]), _.fromPairs)(keys)
```

Comment:
Could it be

```suggestion
export const enumify = _.flow(_.map(key => [key, key]), _.fromPairs)
```

?

Also, it may be good to add a comment with the input type/shape that this function is meant to take.

Reply:
I thought the named `keys` parameter gave more of a hint about the expected input. Replaced that with JSDoc in [5c7968e](https://github.com/DataBiosphere/terra-ui/pull/3316/commits/5c7968e3875664c7c5b3e24497c9817b8374b3cb).